Makefile

```diff
@@ -41,6 +41,10 @@ bf_simple2_yk.o: bf_simple2_yk.c
 bf_simple2_yk: bf_simple2_yk.o
 	clang ${YK_LDFLAGS} ${LDFLAGS} -o $@ $<
 
+.PHONY: check
+check: all
```

Comment:
If this is going to do `cargo test` then we should probably call the target `test`?

Reply:
`check` is the standard name for a test target it make. If you like we can have both

Reply:
I know some people use that name but it seems, at best, very old fashioned -- and isn't very universal (e.g. OpenBSD's framework has `make test` but not `make check`). I'd prefer we stick with `cargo`'s terminology, since we're more familiar with that.

Reply:
unix is old fashioned!

OK. Fixed in 425f49f87 :)

Reply:
I'm going to be an arse: can we get rid of `check` please and just have `test`?

Reply:
Fixed in 375d0e1

Reply:
Alas, poor check, I knew it well :p